specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,9 +6,19 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release is 2022-02-01-preview
+The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
 
-The current stable release of QuestionAnswering is 2021-10-01 and preview release of Luis Deepstack is 2021-07-15-preview
+The current stable release of Question Answering is 2021-10-01.
+
+```yaml
+tag: release_2022_03_01_preview
+add-credentials: true
+clear-output-folder: true
+openapi-type: data-plane
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+```
```

Comment:
These are the defaults but keep getting added to the readme.md. Only one of these blocks should be there. Please remove the other versions that look like this.